test/uk/gov/ons/sbr/models/edit/ReadsOperationSpec.scala

```diff
@@ -0,0 +1,75 @@
+package uk.gov.ons.sbr.models.edit
+
+import org.scalatest.{ FreeSpec, Matchers }
+import play.api.data.validation.ValidationError
+import play.api.libs.json._
+import uk.gov.ons.sbr.models.edit.OperationTypes.{ Replace, Test }
+
+class ReadsOperationSpec extends FreeSpec with Matchers {
```

Comment:
Presumably copied from sbr-control-api and not required in sbr-api?
Same goes for ReadsOperationType, ReadsPatch etc.

What is missing is that sbr-api should have tests to prove that it can write these things correctly ...

Reply:
That's correct, I'll add a `Writes` spec.